homeassistant/data_entry_flow.py

```diff
@@ -47,6 +47,7 @@ def async_progress(self) -> List[Dict]:
             'flow_id': flow.flow_id,
             'handler': flow.handler,
             'source': flow.source,
+            'context': flow.context,
```

Comment:
Why expose this?

Reply:
Basically same purpose as `source`. In trusted networks auth provider, need to verify current IP address is same as context. I can remove above `source` exposure.